module/dialogs/FightDialog.ts

```diff
@@ -111,19 +111,19 @@ export class FightDialog extends ExtendedTestDialog<FightDialogData> {
             this._toggleHidden(e.target);
         });
         html.find('.fighters-grid input, .fighters-grid select').on('change', (e: JQuery.ChangeEvent) => this.updateCollection(e, this.data.data.participants));
-
-        html.find('button[data-action="rollSpeed"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "speed"); });
-        html.find('button[data-action="rollPower"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "power"); });
-        html.find('button[data-action="rollAgility"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "agility"); });
-        html.find('button[data-action="rollSkill"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "skill"); });
-
-        html.find('img[data-action="openSheet"]').on('click', (e: JQuery.ClickEvent) => {
-            const id = e.target.dataset.actorId || "";
-            game.actors.find(a => a._id === id).sheet.render(true);
+        ["Speed", "Power", "Agility", "Skill", "Steel"].forEach((attr: string) => {
+            html.find('button[data-action="roll'+attr+'"]')
+                .on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, attr.toLowerCase() as constants.FightAttr); });
         });
+        const openSheet = (e: JQuery.ClickEvent) => {
+            const id = e.currentTarget.attributes.getNamedItem("data-actor-id").nodeValue || "";
+            game.actors.find(a => a._id === id).sheet.render(true);
+        };
+        html.find('img[data-action="openSheet"]').on('click', openSheet);
+        html.find('div[data-action="openSheet"]').on('click', openSheet);
```

Comment:
```suggestion
        html.find('div[data-action="openSheet", img[data-action="openSheet"]').on('click', (e: JQuery.ClickEvent) => {
            const id = e.currentTarget.attributes.getNamedItem("data-actor-id").nodeValue || "";
            game.actors.find(a => a._id === id).sheet.render(true);
        });
```
You can condense the selector to just one line here

Reply:
@StasTserk for some truly diabolical reason suggested change alone makes Fight! dialog have full screen width on open...

Reply:
Missing left square bracket after the first part of the search.
![image](https://user-images.githubusercontent.com/5915880/104636109-1559fc00-5660-11eb-9de3-b88205f47ee0.png)